x-pack/plugins/features/server/oss_features.ts

```diff
@@ -518,6 +518,31 @@ export const buildOSSFeatures = ({
         },
       },
     },
+    {
+      id: 'savedQueryManagement',
+      name: i18n.translate('xpack.features.savedQueryManagementFeatureName', {
+        defaultMessage: 'Saved Query Management',
+      }),
+      order: 1750,
+      category: DEFAULT_APP_CATEGORIES.management,
+      app: ['kibana'],
+      catalogue: [],
+      privilegesTooltip: i18n.translate('xpack.features.savedQueryManagementTooltip', {
+        defaultMessage:
+          'Set to "All" to enable saving queries across Kibana. Otherwise, set to "None" and configure individually via other feature privileges.',
```

Comment:
Something similar to the second variation would be my preference since it makes it clear another app controls the privilege. Maybe along the lines of this: `If set to "All", saved queries can be managed across Kibana in all applications that support them. If set to "None", saved query privileges will be determined independently by each application.`

Reply:
+1  for `If set to "All", saved queries can be managed across Kibana in all applications that support them. If set to "None", saved query privileges will be determined independently by each application.`

Reply:
@davismcphee @kertal It sounds good to me too! Updated via https://github.com/elastic/kibana/pull/166937/commits/517f11160350e8f72327f79ca51b573d1f706011 Thanks!

<img width="600" alt="Screenshot 2023-09-27 at 13 55 54" src="https://github.com/elastic/kibana/assets/1415710/8657ec51-2ed9-4ce5-9602-b90e964173aa">